app/Console/UpdateUniqueOrganizers.php

```diff
@@ -0,0 +1,179 @@
+    {
+        parent::__construct();
+        $this->connection = $connection;
+    }
+
+    protected function configure(): void
+    {
+        $this->setName('organizer:update-unique')
+            ->setDescription('Updates the table with organizer unique websites based on the `OrganizerCreated` and `WebsiteUpdated` events inside the event store.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $organizerEventsCount = $this->getAllOrganizerEventsCount();
+
+        if ($organizerEventsCount <= 0) {
+            $output->writeln('No `OrganizerCreated` or `WebsiteUpdated` events found.');
+            return 0;
+        }
+
+        $helper = $this->getHelper('question');
+        $updateQuestion = new ConfirmationQuestion('Update ' . $organizerEventsCount . ' organizer(s)? [y/N] ', false);
+        if (!$helper->ask($input, $output, $updateQuestion)) {
+            return 0;
+        }
+
+        $progressBar = new ProgressBar($output, $organizerEventsCount);
+
+        $messages = [];
+        $offset = 0;
+        do {
+            $organizerEvents = $this->getAllOrganizerEvents($offset);
+
+            foreach ($organizerEvents as $organizerEvent) {
+                $organizerUuid = $this->getOrganizerUuid($organizerEvent);
+                $organizerUrl = $this->getOrganizerWebsite($organizerEvent);
+
+                try {
+                    $this->updateOrganizer($organizerUuid, $organizerUrl);
+                    $messages[] = 'Added organizer ' . $organizerUrl . ' with uuid ' . $organizerUuid->toString();
+                } catch (UniqueConstraintViolationException $exception) {
+                    $messages[] = 'Unique exception for organizer ' . $organizerUrl . ' with uuid ' . $organizerUuid->toString();
+                }
+
+                $progressBar->advance();
+            }
+
+            $offset += count($organizerEvents);
+        } while ($offset < $organizerEventsCount);
+
+        $progressBar->finish();
+        $output->writeln('');
+
+        $reportQuestion = new ConfirmationQuestion('Dump update report? [y/N] ', false);
+        if (!$helper->ask($input, $output, $reportQuestion)) {
+            return 0;
+        }
+
+        $reportFile = fopen('update_unique_organizers_report.txt', 'wb');
+        foreach ($messages as $message) {
+            fwrite($reportFile, $message. \PHP_EOL);
+        }
+        \fclose($reportFile);
+
+        return 0;
+    }
+
+    private function getAllOrganizerEvents(int $offset): array
+    {
+        return $this->connection->createQueryBuilder()
+            ->select('uuid, payload')
+            ->from('event_store')
+            ->where('type = "' . self::ORGANIZER_CREATED . '"')
+            ->orWhere('type = "' . self::ORGANIZER_WEBSITE_UPDATED . '"')
+            ->setFirstResult($offset)
+            ->setMaxResults(self::MAX_RESULTS)
+            ->execute()
+            ->fetchAll(PDO::FETCH_ASSOC);
```

Comment:
Should we order the results? Are we sure they will always be in the same order when we fetch the 2nd, 3rd, etc set of 1000 results?

Reply:
Good remark! Better to add an `orderBy` to make sure we get all the events.
I will also add an `orderBy` on the query for the labels.